CHANGELOG_v2.md

```diff
@@ -294,7 +294,7 @@ channel OpenChannelSettings
 - Features/Bugs:
   * ChannelList
     * disableUserProfile: boolean
-    * renderUserProfile: React.Component
+    * renderUserProfile: React.ReactElement
 
 ## 1.2.4(Sept 17, 2020) Deprecated
 
```

Comment:
Is better not to do this, because in v2, its still React.Component
And we make this change in v3.1.2/3